src/main/java/mate/academy/controller/InjectController.java

```diff
@@ -11,23 +15,38 @@
 @RequestMapping("/inject")
 public class InjectController {
```

Comment:
Why did you change this class? The task is to create tests.
Same for RoleDao, RoleService.